### DISCUSSION TITLE
FOSWLY Translate API перенесен на другой сервер

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#580] [Обновления]
В последние дни у прошлого хостера постоянно были какие-то проблемы с сетью, поэтому было решено перенести traslate.toil.cc на более стабильный вариант.
 
```
* FOSWLY Translate API - сервис с помощью которого по умолчанию определяется язык видео на YouTube,
так же, данный сервис является ответственным за перевод ошибок от Yandex API на другие языки.
```

![изображение](https://github.com/ilyhalight/voice-over-translation/assets/62353659/8f23c6ce-e231-44ea-a773-04c2a9ef2fd8)

**Comment by NullVerdict** [comment]
Может перейти с POST на GET? Эти запросы можно закэшировать.

**Reply by ilyhalight** [reply]
Можно попробовать, вроде как, дает небольшой прирост
